guides/release/tutorial/14.md

```diff
@@ -0,0 +1,83 @@
+
+<div class="cta">
+  <div class="cta-note">
+    <div class="cta-note-body">
+      <div class="cta-note-heading">Zoey says...</div>
+      <div class="cta-note-message">
+        <p>`@glimmer/component`, or *[Glimmer component](TODO: link to Glimmer component)*, is one of the several component classes available to use. Component classes are a great starting point whenever you want to add behavior to your components. In this tutorial, we will be using Glimmer components exclusively.</p>
+        <p>In general, Glimmer components should be used whenever possible. However, you may also see `@ember/components`, or *[classic components](TODO: link to classic components)*, being used in older apps. You can tell them apart from looking at their import path, which is helpful for looking up the respective documentation, as they have different and incompatible APIs.</p>
+      </div>
+    </div>
+    <img src="/images/mascots/zoey.png" role="presentation" alt="Ember Mascot">
+  </div>
+</div>
+
+Ember will create an *[instance](TODO: link to instance)* of the class whenever our component is invoked. We can use that to store our state:
+
+```js {data-filename="app/components/rental/image.js" data-diff="+4,+5,+6"}
+import Component from '@glimmer/component';
+
+export default class RentalImageComponent extends Component {
+  constructor(...args) {
+    super(...args);
+    this.isLarge = false;
+  }
+}
+```
+
+Here, in the *[component's constructor](todo://link to component's constructor)*, we *[initialized](TODO: link to initialized)* the *[instance variable](TODO: link to instance variable)* `this.isLarge` to the value `false`, since this is the default state that we want for our component.
+
+Let's update our template to use this state we just added:
+
+```handlebars {data-filename="app/components/rental/image.hbs"}
+{{#if this.isLarge}}
+  <div class="image large">
+    <img ...attributes>
+    <small>View Smaller</small>
+  </div>
+{{else}}
+  <div class="image">
+    <img ...attributes>
+    <small>View Larger</small>
+  </div>
+{{/if}}
+```
+
+The `{{#if ...}}...{{else}}...{{/if}}` *[conditional](TODO: link to conditional)* syntax allows us to render different content. We also have access to the component's instance variables from the template. Combining these two features, we can render either the small or the large version of the image accordingly.
+
+We can verify this is working by temporarily changing the initial value in our JavaScript file. If we change `app/components/rental/image.js` to initialize `this.isLarge = true;` in the constructor, we should see the large version of the property image in the browser. Cool!
+
+<!-- TODO: screenshot? -->
+
+Once we've tested this out, we can change `this.isLarge` back to `false`.
+
+Since this pattern of initializing instance variables from the constructor is pretty common, there happens to be a much more concise syntax for it:
```

Comment:
```suggestion
Since this pattern of initializing instance variables in the constructor is pretty common, there happens to be a much more concise syntax for it:
```